app/controllers/api/services_controller.rb

```diff
@@ -48,7 +48,7 @@ def create
       onboarding.bubble_update('api')
       redirect_to admin_service_path(@service)
     else
-      flash.now[:error] = flash_message(@service.errors.keys.first, {scope: %i[flash services create errors]})
+      flash.now[:error] = @service.errors.full_messages.to_sentence.presence || I18n.t!('flash.services.create.errors.default', {resource_type: product_or_service_type})
```

Comment:
@didierofrivia `@service.errors.full_messages.to_sentence.presence` fetches them all and makes it a sentence, but alternatively you can also pick only the first one by `@service.errors.full_messages.first.presence`

Reply:
Ok, cool! Looks good to me